src/test/regress/expected/multi_create_table.out

```diff
@@ -466,3 +466,52 @@ select create_reference_table('temp_table');
 ERROR:  cannot distribute a temporary table
 DROP TABLE temp_table;
 DROP TABLE shard_count_table_3;
+-- test shard count 1 placement with colocate none.
+-- create a base table instance
+set citus.enable_single_shard_table_multi_node_placement to on;
+CREATE TABLE shard_count_table_1_inst_1 (a int);
+SELECT create_distributed_table('shard_count_table_1_inst_1', 'a', shard_count:=1, colocate_with:='none');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- create another table with similar requirements
+CREATE TABLE shard_count_table_1_inst_2 (a int);
+SELECT create_distributed_table('shard_count_table_1_inst_2', 'a', shard_count:=1, colocate_with:='none');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- Now check placement:
+SELECT (SELECT colocation_id FROM citus_tables WHERE table_name = 'shard_count_table_1_inst_1'::regclass) != (SELECT colocation_id FROM citus_tables WHERE table_name = 'shard_count_table_1_inst_2'::regclass);
```

Comment:
This is not testing the intended thing afaict. You want to test that the placement of the shard is different, not that the colocation id is different. The colocation id would also be different if `citus.enable_single_shard_table_multi_node_placement` was set to `off`.

The placement of the shards can be checked easily in `pg_dist_shard_placement` or `citus_shards`.